controllers/host/host_controller.go

```diff
@@ -1187,6 +1199,41 @@ func hasAdminNetworkChange(profileInterfaces *starlingxv1.InterfaceInfo, current
 			return &profileEth, containsAdminPlatformNetwork(*profileEth.PlatformNetworks)
 		}
 	}
+
+	return nil, false
+}
+
+func hasVLANAdminNetworkChange(profileInterfaces, currentInterfaces *starlingxv1.InterfaceInfo) (*starlingxv1.VLANInfo, bool) {
```

Comment:
Can we add simple UTs against these new methods?

Reply:
ack